lib/plugins/html/index.js

```diff
@@ -20,6 +20,58 @@ const defaultConfig = {
       'transferSize.css',
       'transferSize.image',
       'score.performance'
+    ],
+    summaryBoxes: [
```

Comment:
I think you told me, but I forgot (sorry), is there any reason why we can't use the friendly names here? I know we will miss out on metrics, but maybe we just need to add more data to the friendly list (or just skip a couple of metrics)? I'm thinking it will make sense in a way that we have one way when configuring metrics.

Reply:
Let me try with existing friendlynames. it does have same path structure.